recipes/boost-multi/all/conanfile.py

```diff
@@ -0,0 +1,59 @@
+from conan import ConanFile
+from conan.tools.files import get, copy
+from conan.tools.build import check_min_cppstd
+from conan.tools.layout import basic_layout
+from conan.tools.scm import Version
+from conan.errors import ConanInvalidConfiguration
+import os
+
+
+class MultiConan(ConanFile):
+    name = "boost-multi"
+    homepage = "https://gitlab.com/correaa/boost-multi"
+    description = "Multidimensional array access to contiguous or regularly contiguous memory. (Not an official Boost library)"
+    topics = (
+        "array",
+        "multidimensional",
+        "library",
+    )
+    license = "BSL-1.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    package_type = "header-library"
+    no_copy_source = True
+
+    @property
+    def _min_cppstd(self):
+        return 17
+
+    @property
+    def _compilers_minimum_version(self):
+        return {"gcc": "8",
+                "clang": "7",
+                "apple-clang": "12",
+                "Visual Studio": "16",
+                "msvc": "192"}
+
+    def layout(self):
+        basic_layout(self, src_folder="src")
+
```

Comment:
```suggestion

    def package_id(self):
        self.info.clear()

```

So we remove settings from the package ID.